### ISSUE TITLE
Anomaly: Disguised planet

### ISSUE THREAD

**Issue by ViolentBeetle** [MoreEventsMod/More_Events_Mod_Beta#435]
Level 3 toxic world anomaly. Triggers once per player.

**Initial description:**
(Needs a good name)
Atmospheric patterns of [Planet] do not entirely match simulations, and several deep scans provided data wildly inconsistent with the rest.

**First resolution: Holographic disguise**
(Double the weight)
Further investigation has revealed that supposedly toxic atmosphere of [Planet] does not actually exists. A clever use of holographic generators, light refraction and particle dispersion had created a convincing illusion that the planet has a thick hostile atmosphere.

Once the fake clouds have parted, [Science Ship] had discovered... a completely barren world, concealed beneath the illusion. It seems, someone had gone to a great length to disguise this inhospitable rock as a different kind of inhospitable rock.

>What was even the point?
Planet gains "Holographic projectors" modifier (Ancient holographic projectors produce false image of toxic clouds over this world) and 4 phyisics deposit.

**Second resolution: A Hidden World**
[Science Ship] reports that supposedly toxic atmosphere of [Planet] was not toxic at all. 

A clever use of holographic generators, light refraction and particle dispersion had created a convincing illusion that the planet has a thick hostile atmosphere, while in reality the atmosphere appears to be quite benign. Once the fake clouds have parted, a habitable surface was revealed beneath. 

It seems, at some point someone had gone to quite a length to disguise this world as something far less hospitable

>Fascinating
Planet changes its class: 10% each regular world and 5% for gaia and for tomb. Gains the similarly titled modifier but this version gives +5% to all physics and engineering. Orbital resource tile gets 4 physics deposit. Not to forget to re-roll planet, so it has deposits and stuff.

**Third resolution: Mystery Wrapped in an Enigma**
Further investigation has revealed that supposedly toxic atmosphere of [Planet] does not actually exists. A clever use of holographic generators, light refraction and particle dispersion had created a convincing illusion that the planet has a thick hostile atmosphere.

What [Science Ship] actually discovered beneath beneath the fake clouds was quite odd - a tiny planet covered entirely in a shield, The shield appears to be completely impenetrable, so it remains completely unclear who and for what purpose had decided to not only encase a planet in a shield, but cover it up with fake atmosphere as well...

>Very odd.
Basically, the same as with first option, except the planet is also set to be tiny and shielded.

**Failure:**
[Science Ship] could not replicate previous anomalous readings. Most likely, they were caused by improperly calibrated sensors.
>Disappointing

**Comment by ViolentBeetle** [comment]
I didn't notice your assignments. I did it myself.

It's better to announce this stuff in chats.
